maven-plugins/enforcer-maven-plugin/src/it/projects/example/pom.xml

```diff
@@ -0,0 +1,73 @@
+<!--
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.helidon.build-tools.enforcer.tests</groupId>
+    <artifactId>unpack</artifactId>
+    <version>@project.version@</version>
+    <name>Test enforcer</name>
+    <packaging>pom</packaging>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.helidon.build-tools</groupId>
+                <artifactId>helidon-enforcer-plugin</artifactId>
+                <version>${project.version}</version>
+                <configuration>
+                    <rules>
+                        <rule>inclusive-naming</rule>
+                    </rules>
+                    <inclusiveNamingConfig>
+                        <failOnError>true</failOnError>
+                        <excludes>
```

Comment:
Can you add files to the test example project that would match the configuration ?
E.g.
- test that the excluded files contain offensive word
- test that the excluded terms are not enforced etc
- test that the additional terms are enforced

Reply:
I have added some. It fails with the next (if you set the flag failOnError = true):

-- enforcer results
ERROR: Rule inclusiveNaming failed. Errors:
ERROR:   maven-plugins/enforcer-maven-plugin/target/it/projects/example/src/main/resources/additionalTerm.txt:1: inclusive naming found: [{package:  Recommended replacements[box, bag]}]
ERROR:   maven-plugins/enforcer-maven-plugin/target/it/projects/example/src/main/resources/default.txt:1: inclusive naming found: [{master-slave: Adopt immediately Recommended replacements[Control plane/control plane node, Controller/doer, Primary/replica, Primary/secondary, Leader/follower, Parent/child]}]